keylime/tpm_abstract.py

```diff
@@ -401,7 +399,7 @@ def init_system_rand(self):
                     # as fp has a method fileno(), you can pass it to ioctl
```

Comment:
The code above is broken for py3: `t = struct.pack("ii%ds"%len(rand_data), 8, len(rand_data), str(rand_data))`

`pack` expects binary instead of a string for rand_data, so we need to remove the `str` conversion. 

````python
t = struct.pack("ii%ds"%len(rand_data), 8, len(rand_data), rand_data)
````

Reply:
thanks